README.md

```diff
@@ -1 +1,29 @@
 # prometheus-intro
+Introduction to the prometheus monitoring system
+
+1. Clone this repo
+
+```
+git clone https://github.com/lsst-sqre/prometheus-intro.git
+```
+
+2. Create a virtualenv
+
+```
+cd prometheus-intro
+
+virtualenv prometheus-intro -p python3
+source prometheus-intro/bin/activate
+pip install -r requirements.txt
+```
+
+3. Using the virtualenv in the Jupyter notebook
+
+```
+python -m ipykernel install --user --name=prometheus-intro
```

Comment:
This way I install the python environment in the notebook, so that the python packages installed in the virtualenv environment are also available in the notebook.